internal/openshift/route/const.go

```diff
@@ -3,4 +3,6 @@ package route
 const (
 	// AnnotationIPWhitelist used when for skipping routes.
 	AnnotationIPWhitelist = "haproxy.router.openshift.io/ip_whitelist"
+	// NewRelicStatus used for toggling on/off New Relic Synthetics monitor.
+	NewRelicStatus = "one.newrelic.com/synthetics-status"
```

Comment:
NewRelicStatusUrl sounds better descriptive variable name.

Reply:
It's not a url.. it's a qualifier for an annotation. Can see how it's confusing.